ext/session/session.c

```diff
@@ -2376,6 +2380,31 @@ static PHP_FUNCTION(session_unset)
 }
 /* }}} */
 
+/* {{{ proto int session_gc(void)
+   Perform GC and return number of deleted sessions */
+static PHP_FUNCTION(session_gc)
+{
+	zend_long num;
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	if (PS(session_status) != php_session_active) {
+		php_error_docref(NULL, E_WARNING, "Session is not active");
```

Comment:
I'll document "session_gc() is supposed to be called from periodical task manager such as cron". 

IMHO, users shouldn't care extra new session ID created by session_gc() because many useless session IDs are created by cookie disabled browsers anyway, for example.